### PR TITLE
Prychantovskyi_task1&task2

### PR DESCRIPTION
**Pull request by Prychantovskyi**

### CODE REVIEW

**Review by Dmitriy-NK — COMMENTED** [review]
1. Просьба привести в порядок отображение секции с 5 графиками. Также 3 график с распределением по классам неправилен.
2. Приведите в порядок секцию "Break the previous graph down by gender" там большое количество ошибок в описании, подписях, использовании  plt.subplot2grid((2,3),(1,2)) вместо обычного построения и очень уместно добавление палитры цветов. Также отсутствует график нормированного(в долях единицы) распределения выживших в двух подмножествах 'female','male'. И просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally" в том же разделе
3. В следующем разделе задания (Pclass structure) была просьба построить данные для 2 подмножеств (объединенный 1 и 2 класс) и 3 класс. Также добавьте график с учетом значения Выживших и нет в данных подмножествах и несколько предложений по своей оценке полученных данных.
4. Внимательней проанализируйте данные тепловой карты для confusion matrix, они расходятся с датасетом. по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной (это связано с возможностью классификации с многими метками) так собственный порядок можно задать в аргументе labels, в нашем случае [1,0]  [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
5. Здесь "model.score(x_test, y_test)" данные приводятся без описания и ограничения по количеству знаков после запятой.
6. Для построения ROC Кривых использованы некорректные данные. Данный момент поднимался во время практикума в задании с ирисами. Для построения соответствующего графика нужны данные из predict_proba метода

**Comment by Prychantovskyi** [comment]
Обновил вчера файл, постарался исправить все ошибки. Благодарю, за проверку.

**Review by Dmitriy-NK — COMMENTED** [review]
к 2-му пункту  еще один график- как [пример](https://github.com/srk-practicum/2022-knu-yolo/issues/35)
По 4-му - теперь метки соответствуют значениям но просьба поменять вывод с 
  |   | Predict |   
-- | -- | -- | -- 
  |   | Non-Survive | Survive
Actual | Non-Survive | .. | ..
  | Survive | .. | ..

К такому:
  |   | Predict |   
-- | -- | -- | -- 
  |   | Survive | Non-Survive
Actual | Survive | .. | ..
  | Non-Survive | .. | ..
А также нормировать значения (в долях единицы)

к 5-му - предпочтительнее использовать обозначение Accuracy

**Comment by Prychantovskyi** [comment]
Добрый день, вроде теперь точно все ошибки исправил.

**Review by Dmitriy-NK — COMMENTED** [review]
Вы поменяли подписи но не изменили значения. Внимательней прочитайте 4 пункт, там содержится решение вопроса. Для само проверки можно использовать тот факт что значение Recall для метки "1"(Survived)  должно соответствовать таковому в верхнем левом углу матрицы на пересечении Survived-Survived

**Comment by Prychantovskyi** [comment]
Долго не мог понять почему labels меняют не только название, но потом вчитался в документацию и всё понял. Спасибо за уточнение. Надеюсь теперь всё корректно.

**Review by Dmitriy-NK — APPROVED** [review]
Ок, задание принято.
Перепроверьте, использована одна и та же матрица для всех моделей.

**Comment by Prychantovskyi** [comment]
Добавил второй таск

**Review by Dmitriy-NK — APPROVED** [review]
Task 2 approved
